wiki/People/fr.md

```diff
@@ -0,0 +1,16 @@
+# L'Équipe
+
+*For a list of promotions and retirements, see: [Staff Log](/wiki/People/Staff_Log)*
```

Comment:
```suggestion
*Afin de trouver la liste des promotions et des retraites, veillez consulter : [Archives de l'équipe](/wiki/People/Staff_Log)*
```

Reply:
Just need to be accepted and that's merged !

Reply:
thank you